### ISSUE TITLE
How should the initialScreen parameter be handled

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/joist#717]
For https://github.com/phetsims/QA/issues/649. Related to https://github.com/phetsims/joist/issues/714. Seen on Win 10 Chrome.
The `initialScreen=X` query parameter seems to be handled differently for different simulations. For older sims, such as Acid-Base solutions, it highlights the X option on the home screen, and if X is invalid the sim starts with nothing highlighted but the sim is usable. In other sims, such as Area Model Introduction, it opens the sim to X page and if X is invalid the sim won't load and has a large number of console errors.

![whatshouldinitialscreendo](https://user-images.githubusercontent.com/41024075/118337048-26146480-b4d0-11eb-8f5f-56801e0a62fa.gif)
![whatshouldbadinitialscreendo](https://user-images.githubusercontent.com/41024075/118337157-6673e280-b4d0-11eb-9f88-7c9f0e9b9c6c.gif)

**Comment by KatieWoe** [comment]
Same difference can be seen with invalid screens inputs.

**Comment by jonathanolson** [comment]
Auto-scanning sim differences with the following:

```js
const Maintenance = require( './js/common/Maintenance' );
const ReleaseBranch = require( './js/common/ReleaseBranch' );
const puppeteerEvaluate = require( './js/common/puppeteerEvaluate' );
const checkoutTarget = require( './js/common/checkoutTarget' );
const winston = require( 'winston' );

winston.default.transports.console.level = 'error';

( async () => {
  const releaseBranches = await ReleaseBranch.getMaintenanceBranches();

  for ( let releaseBranch of releaseBranches ) {
    console.log( `${releaseBranch.repo} ${releaseBranch.branch}` );
    try {
      await checkoutTarget( releaseBranch.repo, releaseBranch.branch, false );

      const numScreens = await puppeteerEvaluate( `http://localhost/${releaseBranch.repo}/${releaseBranch.repo}_en.html?brand=phet&ea`, () => {
        return phet.joist.sim.screens.filter( screen => screen !== phet.joist.sim.homeScreen ).length;
      }, {
        waitAfterLoad: 20000
      } );

      console.log( `  screens: ${numScreens}` );

      if ( numScreens >= 2 ) {
        const selectedScreenIndex = await puppeteerEvaluate( `http://localhost/${releaseBranch.repo}/${releaseBranch.repo}_en.html?brand=phet&ea&initialScreen=2`, () => {
          const currentScreen = phet.joist.sim.currentScreenProperty ? phet.joist.sim.currentScreenProperty.value : phet.joist.sim.screenProperty.value;

          const allScreens = [ phet.joist.sim.homeScreen, ...phet.joist.sim.screens.filter( screen => screen !== phet.joist.sim.homeScreen ) ];

          return Math.max( allScreens.indexOf( currentScreen ), 0 );
        }, {
          waitAfterLoad: 20000
        } );

        console.log( `  selected: ${selectedScreenIndex}` );
      }

      // await checkoutTarget( releaseBranch.repo, 'master', false );
    }
    catch ( e ) {
      console.log( e );
    }
  }
} )();
```

**Comment by jonathanolson** [comment]
I believe this is handled in general for the "valid" cases. For "invalid" cases, I'm personally OK with the inconsistency, since master will hard-error an invalid input (without the sim launching).

**Comment by KatieWoe** [comment]
@kathy-phet I noticed a bit of an inconsistency with the initialScreen parameter and @jonathanolson asked me to ask your opinion on it.
I saw this in CCK DC Virtual Lab at least, but I'm not sure how many sims it may impact. https://phet-dev.colorado.edu/html/circuit-construction-kit-dc-virtual-lab/1.2.3-rc.1/phet/circuit-construction-kit-dc-virtual-lab_en_iframe_phet.html. When using `?initialScreen=0` you get a loaded sim with a warning dialog. However, if it is something like `?initialScreen=2` the sim does not load and you get a console error. The errors were what we expected, but the dialog seems like a better option.
<img width="960" alt="initial0" src="https://user-images.githubusercontent.com/41024075/121251628-53042f00-c864-11eb-8ebc-c530c95a506d.png">
<img width="959" alt="initial2" src="https://user-images.githubusercontent.com/41024075/121251868-9363ad00-c864-11eb-86c7-917eb836eb9a.png">
